### PR TITLE
feat: external preset

### PR DESCRIPTION
**Pull request by kpanot**
## Related issues

- :rocket: Feature #719

### CODE REVIEW

File: packages/@o3r/core/schematics/shared/presets/external.preset.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import type { PresetFactory } from './preset.interface';
+import {
+  getAvailableModules,
+  getWorkspaceConfig,
+  OTTER_MODULE_PREFIX_PRESET,
+  OTTER_MODULE_SUPPORTED_SCOPES
+} from '@o3r/schematics';
+
+/**
+ * Preset Installing the minimum modules to fully administrated the application via CMS
```
Comment (fpaul-1A) [inline packages/@o3r/core/schematics/shared/presets/external.preset.ts:12]: doesn't seem accurate

Comment (kpanot) [reply packages/@o3r/core/schematics/shared/presets/external.preset.ts:12]: oops :)

File: packages/@o3r/core/schematics/shared/presets/external.preset.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import {
+  getAvailableModules,
+  getWorkspaceConfig,
+  OTTER_MODULE_PRESET_PREFIX,
+  OTTER_MODULE_SUPPORTED_SCOPES
+} from '@o3r/schematics';
+
+/**
+ * Generate the Preset runner for external presets
+ * @param options
```
Comment (vscaiceanu-1a) [inline packages/@o3r/core/schematics/shared/presets/external.preset.ts:13]: no options param

**Review by matthieu-crouzet — APPROVED** [review]

**Review by fpaul-1A — APPROVED** [review]

**Review by vscaiceanu-1a — APPROVED** [review]

File: packages/@o3r/core/README.md [hunk 3]
```diff
@@ -116,3 +116,15 @@ It includes the following modules:
...
+
+### External (Custom) Preset
+
+The preset mechanism is supporting, via the `--external-preset` option, presets defined by module providers themself.
+
+The modules can be regrouped in custom Presets by following the next steps:
+
+* Adding a **keyword**, in the module *package.json*'s `keywords` field, following the pattern `otter-preset-<my-preset>`.
+* The module needs to be published to public (or private) NPM Registry.
+* The package's scope should be whitelisted, via a Pull Request, in the following [OTTER_MODULE_SUPPORTED_SCOPES constant](https://github.com/AmadeusITGroup/otter/blob/main/packages/@o3r/schematics/src/modules/modules.constants.ts).
```
Comment (cpaulve-1A) [inline packages/@o3r/core/README.md:128]: I know we already discussed about this one, but that means that as long as we have whitelisted them, they can push for any repository on their side without any approval from us ? Is this a good idea? 
I would at least put a disclaimer in the case there are packages that have not been published under our scope.

**Pull request merged**